src/main/kotlin/com/vanniktech/maven/publish/MavenPublishPom.kt

```diff
@@ -29,9 +29,9 @@ internal data class MavenPublishPom(
   internal companion object {
     @JvmStatic
     fun fromProject(project: Project) = MavenPublishPom(
-        project.findMandatoryProperty("GROUP"),
-        project.findMandatoryProperty("POM_ARTIFACT_ID"),
-        project.findMandatoryProperty("VERSION_NAME"),
+        project.findOptionalProperty("GROUP") ?: project.group.toString(),
```

Comment:
Is there internal validation within the constructor that fails on empty gruops / names etc

Reply:
Gradle has a default value for the group if it's not set explicitly: https://docs.gradle.org/current/javadoc/org/gradle/api/Project.html#getGroup--